install_scripts/templates/docker-install/19-03-ce.sh

```diff
@@ -1,10 +1,10 @@
 #!/bin/sh
 set -e
 
-{% if docker_version -%}
+# {% if docker_version -%}
 # this will hardcode the docker version in the script courtesy of replicated
 VERSION="{{ docker_version }}"
-{%- endif %}
+# {%- endif %}
```

Comment:
does commenting this out actually have any effect?

Reply:
only for linter. otherwise its invalid bash